src/main/scala/rocket/ScratchpadSlavePort.scala

```diff
@@ -18,11 +18,7 @@ class ScratchpadSlavePort(address: Seq[AddressSet], coreDataBytes: Int, usingAto
     this(Seq(address), coreDataBytes, usingAtomics)
   }
 
-  val device = new SimpleDevice("dtim", Seq("sifive,dtim0")) {
-    def getMemory(p: DCacheParams, resourceBindings: ResourceBindings): OMDCache = {
-      OMCaches.dcache(p, resourceBindings)
```

Comment:
Is `OMCaches.dcache()` used anywhere anymore? If not, let's delete it.

Reply:
It is used inRocketLogicalTreeNode.